runbooks/how-to-update-for-rda-api-changes.md

```diff
@@ -0,0 +1,53 @@
+# How to Update for RDA API Changes
+
+1. Synch RDA API changes
+   1. Clone/pull latest changes from RDA API repo
+       - https://github.cms.gov/medicare-payment-modernization/rda-api
+   2. Copy changes from RDA API proto files to BFD proto files
+        - RDA API: https://github.cms.gov/medicare-payment-modernization/rda-api/tree/main/proto/src/main/proto/rda/v1
+        - BFD: https://github.cms.gov/medicare-payment-modernization/rda-api/tree/main/proto/src/main/proto/rda/v1
+   3. Update VERSION.txt in BFD repo
+        - https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/proto/rda/VERSION.txt
+2. Update RDA schema/entity mappings
+   1. Diff changes between RDA and BFD proto files
+   2. Make changes to appropriate YAML mapping files
+        - https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-model/bfd-model-rda/mappings
+        - **ALL** properties/classes should be captured, even if ultimately not served to end users.
+        - New properties get added to existing YAML mapping files
+        - New proto files get added as new YAML mapping files
+   4. Create migration file for synchronizing schema with mapping changes
+        - Migrations: https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-model/bfd-model-rif/src/main/resources/db/migration
+        - The [bfd-model-rda](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-model/bfd-model-rda) module can be built to auto-generate a full sql script to aid in migration creation
```

Comment:
This wording confused me.  Maybe `Building the [bfd-model-rda](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-model/bfd-model-rda) module generates a file, `target/generated-sources/entities-schema.sql` containing SQL (both `create table` and `alter column`) that can be used to produce a migration file.